sonar-plugin-api/src/main/java/org/sonar/api/batch/postjob/issue/PostJobIssue.java

```diff
@@ -66,12 +66,12 @@
   String message();
 
   /**
-   * Severity.  Only accurate in preview/issues mode.
+   * Severity.
    */
   Severity severity();
 
   /**
-   * If the issue a new one. Only available in preview/issues mode.
+   * If the issue a new one.
```

Comment:
I would complete with `Tests whether the issue has been introduced during this analysis. {@code false} is returned if the issue is detected as already existing in previous analysis. In this case the line number and line content may have changed during the two analysis.`.